docs/content/src/learn/tutorials/integration-patterns-and-soa/service-orchestration/src/appointment_service/Module.md

```diff
@@ -1,2 +1,44 @@
-# Service Orchestration
+Template for Service Orchestration
 
+# Service Orchestration 
+
+This is a template for the [Service Orchestration](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/integration-patterns-and-soa/service-orchestration/1/). Please refer to it for more details on what you are going to build here. This template provides a starting point for your scenario. 
+
+
+## Using the Template
+
+Run the following command to pull the `appointment_service` template from Ballerina Central.
```

Comment:
Can we change the template name? It's better if the scenario is clear from the template name. Appointment service will not mean anything to someone who sees the title.

Reply:
+1

Reply:
As per the offline discussion the module name, `appointment_service` was kept as it is.